extension/module/module.h

```diff
@@ -38,6 +38,15 @@ class Module {
     MmapUseMlockIgnoreErrors,
   };
 
+  struct MethodHolder {
```

Comment:
can you leave it private and just make a friend class for benchmarking?

Reply:
hmm I don't want to access any data in Module (and don't even have a reference to Module, since benchmarking uses program/method), just want to use MethodHolder as a helper structure. I can also just copy and paste it into the benchmark runner code if you prefer this to be private